samples/maps/demo/all-maps/demo.js

```diff
@@ -2,180 +2,234 @@
-        $('.selector .prev-next').show();
-        $('#side-box').show();
-    }
+    };
 
-    // Is there a layer above this?
-    const match = mapKey
-        .match(/^(countries\/[a-z]{2}\/[a-z]{2})-[a-z0-9]+-all$/);
-    let parent;
-    if (/^countries\/[a-z]{2}\/[a-z]{2}-all$/.test(mapKey)) { // country
-        parent = {
-            desc: 'World',
-            key: 'custom/world'
-        };
-    } else if (match) { // admin1
-        parent = {
-            desc: $('option[value="' + match[1] + '-all.js"]').text(),
-            key: match[1] + '-all'
-        };
-    }
-    $('#up').html('');
-    if (parent) {
-        $('#up').append(
-            $('<a><i class="fa fa-angle-up"></i> ' + parent.desc + '</a>')
-                .attr({
-                    title: parent.key
-                })
-                .on('click', function () {
-                    $('#mapDropdown').val(parent.key + '.js').trigger('change');
-                })
-        );
-    }
+    // On drill up, reset to the top-level map view
+    const afterDrillUp = function (e) {
+        const {
+            mapView,
+            mapName,
+            mapKey
+        } = e.seriesOptions.custom;
+        if (mapView && mapName && mapKey) {
+            fillInfo(mapName, mapKey);
+            input.value = mapName;
 
-    // Data labels formatter. Use shorthand codes for world and US
-    const formatter = function () {
-        return (
-            mapKey === 'custom/world' ||
-            mapKey === 'countries/us/us-all'
-        ) ?
-            (this.point.properties && this.point.properties['hc-a2']) :
-            this.point.name;
+            e.target.mapView.update(
+                Highcharts.merge({
+                    insets: undefined
+                },
+                e.seriesOptions.custom.mapView
+                ),
+                false
+            );
+        }
     };
 
-    // On point click, look for a detailed map to drill into
-    const onPointClick = function () {
-        const key = this.key;
-        $('#mapDropdown option').each(function () {
-            if (this.value === `countries/${key.substr(0, 2)}/${key}-all.js`) {
-                $('#mapDropdown').val(this.value).trigger('change');
+    const data = mapData.objects.default.geometries.map((g, value) => ({
+            key: g.properties['hc-key'],
+            drilldown: g.properties['hc-key'],
+            value
+        })),
+        mapView = Highcharts.merge({
+            projection: {
+                name: 'Miller',
+                rotation: [0]
+            },
+            fitToGeometry: {
+                type: 'MultiPoint',
+                coordinates: [
+                    // Alaska west
+                    [-164, 54],
+                    // Greenland north
+                    [-35, 84],
+                    // New Zealand east
+                    [179, -38],
+                    // Chile south
+                    [-68, -55]
```

Comment:
This was copied over but doesn't look like it was working or now is not working correctly as the initial map seems to be clipped on the side edges.

Reply:
Hmm. funny thing: the current state on the demo pages: https://www.highcharts.com/demo/maps/all-maps The initial map name is `World, Miller projection, medium resolution`, but it is presented in Equal Earth projection, that's why `fitToGeometry` was used. I removed `fitToGeometry` from the code, to keep the cohesion with map name from Map Collection.